Project/DAQ_System/Program/Src/Platform/I2C/Interfaces/ii2c.hpp

```diff
@@ -0,0 +1,32 @@
+/*
+ * ii2c.hpp
+ *
+ *  Created on: Feb 23, 2024
+ *      Author: Nico
+ */
+
+#ifndef II2C_H
+#define II2C_H
+
+#include <cstdint>
+#include <vector>
+
+namespace platform{
+
+class II2C{
+public:
+	static constexpr uint8_t kMaxBytes = 2;
+
+	virtual ~II2C() = default;
+
+	virtual void Master_Receive(uint8_t*) = 0;
+
+	virtual void Master_Transmit(uint8_t*) = 0;
```

Comment:
Can probably simplify to `Receive()` and `Transmit()`

Reply:
🐹 

I too, agree with this sentiment. Obviously, if we wanted to allow for the client device to transmit and receive from the host device this naming might not be as indicative of the function as you probably would have two different receives and 2 different transmits but I digress.